prow/scripts/cluster-integration/helpers/reconciler.sh

```diff
@@ -247,7 +257,7 @@ function reconciler::wait_until_kyma_reconciled() {
 # Deploy test pod
 function reconciler::deploy_test_pod() {
   # Deploy a test pod
-  log::banner "Deploying test-pod in the cluster"
+  log::info "Deploy test pod in the cluster which will trigger reconciliation"
```

Comment:
```suggestion
  log::info "Deploy test-pod in the cluster which will trigger reconciliation"
```
name is important in this case